### PR TITLE
Added '--markup' option.

### PR DESCRIPTION
**Pull request by doloopwhile**
Hello.

We are using pdoc for our internal packages. That's very convenient!

However, Some my coworkers are unwilling for pdoc because pdoc assumes all docstrings are written in Markdown and formats their plain-text docstrings wrongly.

Therefore I added `--style` option:

- `--style=markdown`: to interpret as Markdown (current and default behavior).
- `--style=pre`: just to wrap docstring with \<pre\>.

I bet that `--style` is going to boost the users of pdoc.

I think [Google's style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) and [Numpy's style](https://docs.scipy.org/doc/numpy-1.15.0/docs/howto_document.html#numpydoc-docstring-guide) are also expected by not a few people. I will add this two styles if this pull-request is accepted. (However, Currently, I do not have enough time.)

Thanks.

### CODE REVIEW

**Comment by mhils** [comment]
refs #153

**Comment by doloopwhile** [comment]
thanks.
I am going to fork.

File: pdoc/cli.py [hunk 1]
```diff
@@ -99,7 +100,22 @@
     default=8080,
     help="The port on which to run the HTTP server.",
 )
+aa(
+    "--style",
```
Comment (kernc) [inline pdoc/cli.py]: I would prefer `--docstring-style`. Less ambiguous.

File: pdoc/templates/html_index.mako [hunk 2]
```diff
@@ -8,14 +8,14 @@ import pdoc.html_helpers as hh
...
-<h1>Python module list</h1>
+  <h1>Python module list</h1>
   <table id="module-list">
   % for root in roots:
     <tr>
       <td><a href="${link_prefix}${root.name}">${root.name}</a></td>
       <td>
       % if len(root.docstring.strip()) > 0:
-        <div class="desc">${root.docstring | hh.mark}</div>
+        <div class="desc}">${root.docstring | hh.mark(markup)}</div>
```
Comment (mhils) [inline pdoc/templates/html_index.mako]: typo?

**Pull request closed without merging**